src/app/pages/HomePage/Graph/ParaTimeSelector/index.tsx

```diff
@@ -185,15 +186,21 @@ const ParaTimeSelectorCmp: FC<ParaTimeSelectorProps> = ({ disabled, step, setSte
   }
 
   const onZoomOutClick = () => {
-    setSelectedLayer(Layer.consensus)
+    setSelectedLayer({ current: Layer.consensus })
   }
 
   const onPinchZoom = ({ x, y, scale }: UpdateAction) => {
+    if (step !== ParaTimeSelectorStep.Explore) {
+      return
+    }
+
     const transformValue = make3dTransformValue({ x, y, scale })
     setScale(scale)
     quickPinchZoomInnerRef.current?.style.setProperty('transform', transformValue)
   }
 
+  const isZoomedIn = parseFloat(scale.toFixed(2)) > 1
```

Comment:
How come `toFixed(2)`?

Reply:
This is effectively `scale > 1.005`

Reply:
Initial zoom is something like 1.000000132930. For that purpose I just round to 2 decimals.